### PR TITLE
data: fabian4

### PR DESCRIPTION
**Pull request by fabian4**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5431"></a>

### CODE REVIEW

File: data/fabian4.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "icon": "FaInstagram",
+            "url": "https://www.instagram.com/fabian__bao"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/fabian4"
+        },
+        {
+            "icon": "FaLinkedin",
+            "url": "https://linkedin.com/in/fabian4"
```
Comment (priyankarpal) [inline data/fabian4.json:20]: Your LinkedIn profile link doesn't exist, please add a valid URL

image

File: data/fabian4.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            "icon": "FaTwitter"
+        },
+        {
+            "name": "MyBlog - Follow me on MyBlog",
+            "url": "https://fabian4.site/",
+            "icon": "SiHashnode"
+        },
+        {
+            "name": "LinkedIn - Follow me on LinkedIn",
+            "url": "https://linkedin.com/in/fabian4",
```
Comment (priyankarpal) [inline data/fabian4.json:45]: Here also Please add a valid URL

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/fabian4.json (hunk 1 above)
Comment (fabian4) [reply data/fabian4.json:20]: You can reach it now

File: data/fabian4.json (hunk 2 above)
Comment (priyankarpal) [inline data/fabian4.json:45]: Not working. Please add a valid URL

File: data/fabian4.json (hunk 1 above)
Comment (priyankarpal) [inline data/fabian4.json:20]: Not working. Please add a valid URL 

```
https://linkedin.com/in/fabian4
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**